test/additional-properties.js

```diff
@@ -9,7 +9,7 @@ var expandTypes = require('../lib/raml')
 describe('Additional properties', function () {
   it('should be expanded correctly', function () {
     return require('raml-1-parser')
-      .loadRAML('test/fixtures/additional-properties.raml', {rejectOnErrors: true})
+      .loadRAML('test/fixtures/additional-properties.raml', { rejectOnErrors: true })
```

Comment:
Style changes here (additional whitespaces) and in the rest of the files are not related to the main intention of this PR so it would be better to raise a separate PR for this.